packages/styles/scss/components/radio-button/_radio-button.scss

```diff
@@ -106,7 +102,9 @@ $radio-border-width: 1px !default;
       background-color: $icon-primary;
       border-radius: 50%;
       content: '';
-      transform: scale(0.5);
+      transform: scale(0);
+      transition: background-color $duration-fast-02 motion(exit, productive),
+        transform $duration-moderate-01 motion(exit, productive);
```

Comment:
We recently added an a11y helper to disable animations when `prefers-reduced-motion` is enabled. Can we add this media query to this style block?

```css
      @media (prefers-reduced-motion: reduce) {
        transition: none;
      }
```